packages/server/lib/util/print-run.ts

```diff
@@ -22,6 +22,19 @@ type Screenshot = {
   specName: string
 }
 
+export const cloudRecommendationMessage = `
+-----------------------------------------------------------------------------------------
+
+  Having trouble debugging your CI failures?
+  
+  Record your runs to the Cypress Cloud to watch video recordings for each test, 
```

Comment:
cc @pstakoun, good to get product :eyes:  for any public facing marketing call to action messages.